release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -2,20 +2,42 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io' as io;
+
+import 'package:conductor_core/conductor_core.dart';
+import '../services/conductor.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
 import 'package:flutter/material.dart';
+import 'package:platform/platform.dart';
+import 'package:process/process.dart';
 
+import '../logic/cherrypicks.dart';
 import 'common/tooltip.dart';
 
+const FileSystem fileSystem = LocalFileSystem();
```

Comment:
Do we still need all of these if we have `ConductorService`?

Reply:
Oh my bad, I forgot to remove them.